toolkit/imageconfigs/postinstallscripts/core-container/cleanup.sh

```diff
@@ -7,3 +7,7 @@ rm -rf /boot/*
 rm -rf /usr/src/
 rm -rf /home/*
 rm -rf /var/log/*
+
+echo removing tdnf cache
```

Comment:
There is already a script that does it https://github.com/microsoft/azurelinux/blob/3.0-dev/toolkit/imageconfigs/postinstallscripts/remove-tdnf-cache.sh

But might I suggest imbedding this action at the end of build in our tooling instead? I have a branch that moves the script into our Go tooling itself, I just never got around to testing it. But after some discussion a while ago we determined its better for it to live in our build tooling and not post install scripts

Reply:
Remove-tdnf-cache script is used for VHD/VHDX in images cleanup and this cleanup script is specific to core-conatiner.  I think yes it is good to add  tdnf cache clean-up from toolkit.